units/host_termination.go

```diff
@@ -332,12 +342,18 @@ func (j *hostTerminationJob) Run(ctx context.Context) {
 		attribute.String(evergreen.HostIDOtelAttribute, j.host.Id),
 		attribute.Float64(fmt.Sprintf("%s.idle_secs", hostTerminationAttributePrefix), j.host.TotalIdleTime.Seconds()),
 		attribute.Float64(fmt.Sprintf("%s.running_secs", hostTerminationAttributePrefix), j.host.TerminationTime.Sub(j.host.StartTime).Seconds()),
+		attribute.String(fmt.Sprintf("%s.ip_allocation_id", hostTerminationAttributePrefix), j.host.IPAllocationID),
+		attribute.String(fmt.Sprintf("%s.ip_association_id", hostTerminationAttributePrefix), j.host.IPAssociationID),
+		attribute.String(fmt.Sprintf("%s.started_by", hostTerminationAttributePrefix), j.host.StartedBy),
 		attribute.Bool(fmt.Sprintf("%s.user_host", hostTerminationAttributePrefix), j.host.UserHost),
 		attribute.Int(fmt.Sprintf("%s.task_count", hostTerminationAttributePrefix), j.host.TaskCount),
 	)
 	if !utility.IsZeroTime(j.host.BillingStartTime) {
 		span.SetAttributes(attribute.Float64(fmt.Sprintf("%s.billable_secs", hostTerminationAttributePrefix), j.host.TerminationTime.Sub(j.host.BillingStartTime).Seconds()))
 	}
+	if instanceType != "" {
+		span.SetAttributes(attribute.String(fmt.Sprintf("%s.instance_type", hostTerminationAttributePrefix), instanceType))
+	}
```

Comment:
This LGTM, I was thinking it might be better to just move the instanceType logic down here but actually I think it will make the span code bloated 🤷